validator/lib/isTile.js

```diff
@@ -1,29 +1,41 @@
 'use strict';
 const path = require('path');
 const Cesium = require('cesium');
+const defaultValue = Cesium.defaultValue;
 const defined = Cesium.defined;
 module.exports = isTile;
 
+var emptyArray = [];
+
 /**
  * Checks whether the given file path is a tile file path.
  *
  * @param {String} filePath The file path.
- * @param {String} version The tileset version
+ * @param {Object} [options] Options object
  * @returns {Boolean} True if the file path is a tile file path, false if not.
  */
-function isTile(filePath, version) {
+function isTile(filePath, options) {
     var extension = path.extname(filePath);
-    if (defined(version)) {
-        if (version === '1.0') {
+    if (defined(options)) {
+        var version = options.version;
+        var tileset = options.tileset;
+        var extensionsUsed = defaultValue(tileset.extensionsUsed, emptyArray);
+        var hasContentGltfExtension = extensionsUsed.indexOf('3DTILES_content_gltf') > -1;
+
+        if (version === '2.0.0-alpha.0' || hasContentGltfExtension) {
```

Comment:
Is the `2.0.0-alpha.0` version documented anywhere? It's not in the 3DTILES_content_gltf PR.

Reply:
I added a clarifying comment: https://github.com/CesiumGS/3d-tiles-validator/pull/209/commits/975fab4ac80cd25ec42d9cc6817a5c4488689b1e